projects/storefrontlib/cms-structure/seo/structured-data/json-ld.directive.ts

```diff
@@ -4,8 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
Did you plan to remove the sanitization part from `JsonLdScriptFactory` in another PR?
Or maybe you think it should stay in there?

Reply:
the issue was raised from Checkmarx scan vulnerability about bypassSecurityTrustHtml, so the idea was to make minimal changes for 5.0.x to have clean checkmarx scan.